src/In.ProjectEKA.DefaultHip/Resources/demoPatientCareContextDataMap.json

```diff
@@ -143,7 +143,7 @@
             { 
                 "capturedOn": "2018-02-02",
                 "data":  {
-                    "condition" : ["BI-KTH-12FirstVisitConditions.json"],
+                    "opconsultation" : ["BI-KTH-12FirstVisitConditions.json"],
```

Comment:
don't change the type. its ok to leave it as condition. we will setup data for that differently.

Reply:
Sure, thanks